.venv/bin/DumpNTLMInfo.py

```
@@ -0,0 +1,664 @@
+        if bootTime != 0:
+            print("[+] Boot Time       : {}".format(bootTime))
+            print("[+] Server Up Time  : {}".format(currentTime - bootTime))
+
+
+    def DisplayChallangeInfo(self, challange):
+        if challange['TargetInfoFields_len'] > 0:
+            av_pairs = ntlm.AV_PAIRS(challange['TargetInfoFields'][:challange['TargetInfoFields_len']])
+            if av_pairs[ntlm.NTLMSSP_AV_HOSTNAME] is not None:
+                try:
+                    print("[+] Name            : {}".format(av_pairs[ntlm.NTLMSSP_AV_HOSTNAME][1].decode('utf-16le')))
+                except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)